src/components/MovieDetail/MovieDetail.css

```diff
@@ -2,13 +2,45 @@
   border: solid black 0.1em;
 }
 
-.movie-info {
+.backdrop-img {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 100vh;
+    width: 100vw;
+    
+}
+
+.movie-container {
+  width: 50%;
+  height: 50%;
```

Comment:
This looks good. I think there's a lot of candidates for sass here, so I think we should try to get that going this week- workload permitting.